common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -226,11 +226,11 @@ def _clear_bulk_ops_record(self, course_key):
                 course_key.replace(org=None, course=None, run=None, branch=None)
             ]
 
-    def _start_outermost_bulk_operation(self, bulk_write_record, course_key):
+    def _start_outermost_bulk_operation(self, bulk_write_record, course_key, ignore_case=False):
         """
         Begin a bulk write operation on course_key.
         """
-        bulk_write_record.initial_index = self.db_connection.get_course_index(course_key)
+        bulk_write_record.initial_index = self.db_connection.get_course_index(course_key, ignore_case=ignore_case)
         # Ensure that any edits to the index don't pollute the initial_index
         bulk_write_record.index = copy.deepcopy(bulk_write_record.initial_index)
         bulk_write_record.course_key = course_key
```

Comment:
@cpennington You have raised an interesting point. We are updating the `bulk_operations` method and not `get_course` or `create_course`. I will investigate more about your question.